packages/forms/frontend/sirius-components-widget-reference/src/ReferenceWidgetFragment.types.ts

```diff
@@ -47,51 +46,132 @@ export interface GQLReferenceValue {
   hasClickAction: boolean;
 }
 
-export interface GQLEditReferencePayload {
-  __typename: string;
+export interface GQLErrorPayload extends GQLClickReferenceValuePayload {
```

Comment:
Why doesn't the GQLErrorPayload extend GQLClearReferencePayload (and other payloads too)? It seems clear from your GraphQL schema that clearing a reference can return an error payload.

Reply:
Done